docs/user/security-log.rst

```diff
@@ -1,16 +1,19 @@
 Security Log
 ============
 
-Security logs allow you to see what has happened recently in your organization or account.
-We store the IP address and the browser's User-Agent_ on each event,
+Security logs allow you to audit what has happened recently in your organization or account.
+This feature is quite important for many security compliance programs,
+as well as the general peace of mind of knowing what is happening on your account.
+We store the IP address and the browser used on each event,
 so that you can confirm this access was from the intended person.
```

Comment:
```suggestion
so that you can confirm this access was from the intended person.

Security logs are only visible to organization owners.
You can invite other team members as owners.
```

I was wondering where to include this. It's a detail but it's also "general" so works best in the introduction.